.github/workflows/issue-bot.yaml

```diff
@@ -10,24 +10,24 @@ jobs:
     steps:
       - name: Remove triage label on close
         if: github.event.action == 'closed'
-        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label "status:triage"
+        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label 'status:triage'
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Update issue status when assigned
         if: github.event.action == 'assigned'
-        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --add-label "status:in-progress"
+        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --add-label 'status:in-progress'
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Update issue status when unassigned
         if: github.event.action == 'unassigned'
-        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label "status:in-progress"
+        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label 'status:in-progress'
         env:
           GH_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
       - name: Remove triage label on status change
-        if: github.event.action == 'labeled' && startsWith(github.event.label.name, "status:") && github.event.label.name != "status:triage"
-        run: gh issue edit --repo prefecthq/prefect ${{ github.event.issue.number }} --remove-label "status:triage"
+        if: github.event.action == 'labeled' && startsWith(github.event.label.name, 'status:'') && github.event.label.name != 'status:triage'
```

Comment:
Thanks for fixing it! I think it's _just_ this line. Double quotes are allowed in the `run` commands since they're just in bash. The singe-quote requirement in actions expressions gets me often 😮‍💨

Reply:
Oh yeah, makes sense